README.md

```diff
@@ -5,30 +5,35 @@ cinnamon-desktop contains the libcinnamon-desktop library, the cinnamon-about
 program as well as some desktop-wide documents.
 
 The libcinnamon-desktop library provides API shared by several applications
-on the desktop, but that cannot live in the platform for various
-reasons. There is no API or ABI guarantee, although we are doing our
-best to provide stability. Documentation for the API is available with
-gtk-doc.
+on the desktop, but that cannot live in the platform for various reasons. 
 
+There is no API or ABI guarantee, although we are doing our best to provide stability. 
+
+Documentation for the API is available with gtk-doc.
 
 Installation
 ============
 
 1) Run meson with options you like. The following configuration installs
    all binaries, libs, and shared files into /usr/local, and enables all
    available options:
-
-    meson debian/build \
-        --prefix=/usr/local \
-        --buildtype=debug \
-        -D deprecated_warnings=false
+   
+   ```shell
+    meson debian/build --prefix=/usr/local --buildtype=debug -D deprecation_warnings=false
+   ```
 
 2) Compile and install (sudo is needed for install)
-
-    ninja -C debian/build
-    ninja -C debian/build install
+   
+   ```shell
+   meson build debian/build
+   meson compile -C debian/build
+   meson install -C debian/build
+   ```
```

Comment:
Is there a reason you removed the existing commands and replaced them? The new commands are also duplicative and incorrect. For me, `meson build debian/build` generates errors but this is also duplicative of the command that works from step 1. This is on systems with meson version 0.61.2. Also, `meson compile` and `meson install` both just run the `ninja` commands from the previous commands that were in the documentation. Same for the uninstall steps that come after this. I don't understand the need to obfuscate those commands.


Apologies if there's any confusion on my part.